sparse/tests/test_coo.py

```diff
@@ -85,6 +125,7 @@ def test_ufunc_reductions_kwargs(reduction, kwargs):
 @pytest.mark.parametrize('keepdims', [False])
 @pytest.mark.parametrize('fraction', [0.25, 0.5, 0.75, 1.0])
 @pytest.mark.filterwarnings('ignore:All-NaN')
+@pytest.mark.filterwarnings('ignore:Mean of empty slice')
 def test_nan_reductions(reduction, axis, keepdims, fraction):
     s = sparse.random((2, 3, 4), data_rvs=random_value_array(np.nan, fraction),
```

Comment:
It seems you forgot to change to the fixture here.

Reply:
I don't think so? `nanmean` throws that warning instead of `All-NaN`, we need both filters to cover all the warnings from this test. Unless I'm misunderstanding your comment here.

Reply:
Ah, my bad. The fixture doesn't do NaNs. Ignore. It would be nice to and a non-floating `dtype` here to hit the line we missed -- But not is okay too.

Reply:
I meant that the fixture you created wasn't used in the NaN-reductions, but like I said, you can safely ignore me.